python/src/iceberg/io/base.py

```diff
@@ -76,7 +76,15 @@ def close(self) -> None:
 
 
 class InputFile(ABC):
-    """A base class for InputFile implementations"""
+    """A base class for InputFile implementations
+
+    Args:
+        location(str): A URI or a path to a local file
+
+    Attributes:
+        location(str): The URI or path to a local file for an InputFile instance
+        exists(bool): Whether the file exists or not
```

Comment:
This is no longer an attribute/property right?

Reply:
Nope, I removed it here. I also removed filesystem since that's only used internally and not even set as a property.